charts/document-engine/values.yaml

```diff
@@ -505,6 +505,23 @@ documentConversion:
   # @section -- 9. Document conversion
   spreadsheetMaxContentWidthMm: 0
 
+# -- (object) Clustering settings
```

Comment:
Sorting is screwed, it's plain alphabetical. Look at the readme: section gets on top.
Think where you want it and pick something lower? E.g. `8` (then it will be around digital signatures).